en/02_Developer_Guides/03_Forms/Field_types/04_GridField.md

```diff
@@ -242,24 +239,20 @@ $gridField->setConfig($config);
 
 ### `GridFieldConfig_RecordEditor`
 
+> [!CAUTION]
+> Permission control for editing and deleting the record uses the `canEdit()` and `canDelete()` methods on the class that represents your data.
+
 Similar to `GridFieldConfig_RecordViewer` with the addition support to edit or delete each of the records.
 
-[info]
 Each record in the list must have an `ID` field, and the value of that field must be a positive integer.
 
 If the class representing your data has a `getCMSFields()` method, the return value of that method will be used for the fields displayed in the read-only view.
 Otherwise, you'll need to pass in a [`FieldList`](api:SilverStripe\Forms\FieldList) to [`GridFieldDetailForm::setFields()`](api:SilverStripe\Forms\GridField\GridFieldDetailForm::setFields()).
-[/info]
```

Comment:
Removed from the callout block because that's more suitable for this content, and to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.